### DISCUSSION TITLE
Major dependencies version jump

### DISCUSSION THREAD

**Discussion by tsunhopang** [nuclear-multimessenger-astronomy/nmma#95] [Announcements]
Hi, all nmma-developer and nmma-user,

We have a major dependencies version jump for bilby, bilby-pipe, and parallel-bilby.

Please try to have the new version installed and test it to see if it has any unexpected behavior.

Cheers,
Peter, on behalf of nmma developing team

**Comment by mcoughlin** [comment]
@tylerbarna @bking-astro @shreyasahasram08 It will be worth making a new environment and see if it is easier to install, for example.

**Comment by mcoughlin** [comment]
Also @weizmannk of course.

**Comment by sahiljhawar** [comment]
Hi all,

Anyone who is using `bilby_pipe>=1.1.0` from the recent dependency change may locally change their` eos/create_injection.py`. Since in `bilby_pipe>=1.1.0`, the support for `gps_files` in `InjectionCreator` has been removed and `gpstimes` has been added. Both of them take the same input.

**Reply by mcoughlin** [reply]
@tsunhopang Maybe something to look at?

**Reply by tsunhopang** [reply]
yea I will push a fix for this shortly

**Reply by tsunhopang** [reply]
I have pushed commit 72024218f4dba03bc80d3b8eb283d3890fd88a85, which update the argument for `gps_file` for all instance of `InjectionCreator`
